bundles/org.openhab.binding.hue/README.md

```diff
@@ -117,38 +117,37 @@ Bridge hue:bridge:1 [ ipAddress="192.168.0.64", userName="qwertzuiopasdfghjklyxc
 The devices are identified by the number that the Hue Bridge assigns to them (also shown in the Hue App as an identifier).
 Thus, all it needs for manual configuration is this single value like
 
-```
+```text
```

Comment:
```java ??

Reply:
Your comment does not render well. If I edit it I see you suggest to change this to ```java

Reply:
Yes